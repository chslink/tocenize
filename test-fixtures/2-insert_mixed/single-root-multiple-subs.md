# root heading
blah blah

## 1st sub
blah

## 2nd sub
blah

### subsub