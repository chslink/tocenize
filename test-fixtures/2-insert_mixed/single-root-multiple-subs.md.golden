# root heading
blah blah

- [root heading](#root-heading)
	- [1st sub](#1st-sub)
	- [2nd sub](#2nd-sub)
		- [subsub](#subsub)

## 1st sub
blah

## 2nd sub
blah

### subsub